pkg/services/dashboards/service/dashboard_service_test.go

```diff
@@ -143,153 +143,7 @@ func TestDashboardService(t *testing.T) {
 			_, err := service.SaveDashboard(ctx, dto, false)
 			require.Equal(t, err, dashboards.ErrFolderNotFound)
 		})
-
-		t.Run("Should return validation error if dashboard is provisioned", func(t *testing.T) {
```

Comment:
dashboard provisioning validation is tested here: https://github.com/grafana/grafana/blob/main/pkg/registry/apis/dashboard/register_test.go#L24 and in pkg/tests/apis/dashboard